assets/js/admin/admin.js

```diff
@@ -221,4 +221,61 @@
 		$(this).append(video);
 
 	});
+
+	// Toggles edit state.
+	function toggleEditTitle( e ) {
+		e.preventDefault();
+
+		var $ele = $( '#edit-form-name' ),
+			$input_title = $ele.siblings( '#evf-edit-form-name' );
+		// Toggle disabled property.
+		$input_title.prop( 'disabled' , function(_, val) {
+			return ! val;
+		});
+		if ( ! $input_title.hasClass( 'everst-forms-name-editing' ) ) {
+			$input_title.focus();
+		}
+		$input_title.toggleClass( 'everst-forms-name-editing' );
+	}
+
+	// Delegates event to toggleEditTitle() on clicking
+	$( '#edit-form-name' ).on( 'click', function( e ) {
+		e.stopPropagation();
+		if ( '' !== $( '#evf-edit-form-name').val().trim() ) {
+			toggleEditTitle( e );
+		}
+	});
+
+	// Apply the title change to form name field.
+	$( '#evf-edit-form-name' ).on( 'change', function( e ) {
+		e.preventDefault();
+
+		var $this = $( this );
+		if ( '' !== $this.val().trim() ) {
+			$( '.everest-forms-form-name' ).html( $this.val().trim() );
+			$( '#everest-forms-panel-field-settings-form_title' ).val( $this.val().trim() );
+		}
+	})
+
+	.on( 'keypress', function(e) {
+		e.stopPropagation();
+
+		if ( 13 === e.which && '' !== $( this ).val().trim() ) {
+			toggleEditTitle( e );
+		}
+	})
+
+	.on( 'click', function(e) {
+		e.stopPropagation();
+	});
+
+	// In case the user goes out of focus from title edit state.
+	$(document).not( $( '.everest-forms-title-desc' ) ).click( function( e ) {
+		e.stopPropagation();
+		var field = $( '#evf-edit-form-name' );
```

Comment:
Make sure the variable is declared first, empty/blank line is there before any docblock for neat coding. Additionally, prefer space between statement.

`if ( ! field.pro.....` and `$( document ).not(......`

Reply:
Variables are now declared before event handling.